### ISSUE TITLE
SA1125 UseShorthandForNullableTypes incorrectly reported for static access through Nullable<int>

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#636]
The following expression incorrectly results in SA1125:

```csharp
bool equal = Nullable<int>.Equals(null, null);
```
